File: Examples/CSharp/Programming-Documents/Sections/AppendSectionContent.cs
using System;
using System.Collections.Generic;
using Aspose.Words;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Sections
{
    class AppendSectionContent
    {
        public static void Run()
        {
            //ExStart:AppendSectionContent
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithSections();
            Document doc = new Document(dataDir + "Section.AppendContent.doc");
            // This is the section that we will append and prepend to.
            Section section = doc.Sections[2];

            // This copies content of the 1st section and inserts it at the beginning of the specified section.
            Section sectionToPrepend = doc.Sections[0];
            section.PrependContent(sectionToPrepend);

            // This copies content of the 2nd section and inserts it at the end of the specified section.
            Section sectionToAppend = doc.Sections[1];
            section.AppendContent(sectionToAppend);
            //ExEnd:AppendSectionContent
            Console.WriteLine("\nSection content appended successfully.");
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Sections/AppendSectionContent.vb
Imports Microsoft.VisualBasic
Imports Aspose.Words
Public Class AppendSectionContent
    Public Shared Sub Run()
        ' ExStart:AppendSectionContent
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithSections()
        Dim doc As New Document(dataDir & Convert.ToString("Section.AppendContent.doc"))
        ' This is the section that we will append and prepend to.
        Dim section As Section = doc.Sections(2)

        ' This copies content of the 1st section and inserts it at the beginning of the specified section.
        Dim sectionToPrepend As Section = doc.Sections(0)
        section.PrependContent(sectionToPrepend)

        ' This copies content of the 2nd section and inserts it at the end of the specified section.
        Dim sectionToAppend As Section = doc.Sections(1)
        section.AppendContent(sectionToAppend)
        ' ExEnd:AppendSectionContent
        Console.WriteLine(vbLf & "Section content appended successfully.")
    End Sub
End Class
